test/e2e-cypress/tests/bugs/4641.js

```diff
@@ -0,0 +1,49 @@
+describe("#4641: The Logout button in Authorize popup not clearing API Key", () => {
```

Comment:
I'd like to see a third test case that covers the following scenario.

Given this definition:

```yaml
openapi: 3.0.0
paths:
  /:
    get:
      description: ok
      security:
      - securityA: []
      - securityB: []
      responses: 
        200: 
          description: ok
components:
  securitySchemes:
    securityA:
      type: apiKey
      name: SecurityA
      in: header
    securityB:
      type: apiKey
      name: SecurityB
      in: header
```

1. I authorize securityA and securityB.
2. I logout and then attempt to reauthorize securityB (idea being: the data will be blank, so reauthorization won't happen).
3. I execute my operation.
4. **Assert that securityA was attached to the request, and securityB was not.**

My rationale here is that the data-clearing code is in the Auths component, which has the capability to wipe out more than one authorization's data, so we should have a defensive test against that happening.

Reply:
done in a0b8f4f